tycho-bundles/org.eclipse.tycho.p2.resolver.impl/src/main/java/org/eclipse/tycho/p2/target/TargetPlatformFactoryImpl.java

```diff
@@ -307,28 +311,60 @@ private IMetadataRepository loadMetadataRepository(MavenRepositoryLocation locat
         }
     }
 
+    private static final class SortedRepositories {
```

Comment:
can you add a little comment by what mean it is sorted? I assume its local first, then remote or something?